components/feral-connectd/config.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+	"sync"
+)
+
+// Configuration for CDP connection
+type Config struct {
+	sync.RWMutex
+	CDPHost string
+	CDPPort int
+
+	WsURL    string
+	WsAPIKey string
+
+	LocationID string
+	TopicID    string
+}
+
+// Environment variable names
+const (
+	ENV_CDP_PORT   = "CDP_PORT"
```

Comment:
Should read from file instead of using env. We assume we would use `/var/lib/feralfile/creds.json`.
The format would be 
```
{
    "relayer" : {
        "endpoint" : "",
        "apiKey" : ""
    },
    "cdp" : {
        "endpoint" : ""
    }
}
```